src/controllers/connectionManager.ts

```diff
@@ -616,11 +635,18 @@ export default class ConnectionManager {
         });
     }
 
+    /**
+     * Delete a credential from the credential store
+     */
+    public async deleteCredential(profile: Interfaces.IConnectionProfile): Promise<boolean> {
+        let result = await this._connectionStore.deleteCredential(profile);
```

Comment:
nit - is there a reason you define a var here? If it's for debugging you can see the return value for a function by setting an inline breakpoint on the return statement. Not a huge deal but it's just unnecessary clutter and adds extra work creating the var